PhilipFunkos/src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import "./App.css";
+import Home from "./home";
+import Product from "./products";
+import Contact from "./contact";
+
+function App() {
+  function home() {
+    let page1 = document.querySelector(".home-page");
+    let page2 = document.querySelector(".products-page");
+    let page3 = document.querySelector(".contact-page");
+    page1.classList.remove("disable");
+    page2.classList.add("disable");
+    page3.classList.add("disable");
+  }
+  function product() {
+    let page1 = document.querySelector(".home-page");
+    let page2 = document.querySelector(".products-page");
+    let page3 = document.querySelector(".contact-page");
+    page2.classList.remove("disable");
+    page1.classList.add("disable");
+    page3.classList.add("disable");
+  }
+  function contact() {
+    let page1 = document.querySelector(".home-page");
+    let page2 = document.querySelector(".products-page");
+    let page3 = document.querySelector(".contact-page");
+    page3.classList.remove("disable");
+    page1.classList.add("disable");
+    page2.classList.add("disable");
+  }
```

Comment:
Code duplication on selectors.

Then you just enabled/disable with flags here

You can do something like:

function disablePages(page1Enabled,  page2Enabled, page3Enabled) {
    let page1 = document.querySelector(".home-page");
    let page2 = document.querySelector(".products-page");
    let page3 = document.querySelector(".contact-page");
      enableDisable(page1, true);
      enableDisable(page2, false);
      enableDisable(page3, false);
      

}

function enableDisable(page, isEnabled) {
     if( isEnabled)
          page.classList.remove("disabled");
     else
           page.classList.add("disabled);
     
}

Reply:
All he needs to do this is use react state instead, its way easier.

```
const [page, setPage] = useState('Home')

const handleClick = (page) => {
  setPage(page)
}

...

{page === 'Home' && <Home />}
{page === 'Product' && <Product />}
{page === 'Contact' && <Contact />}

```

Reply:
If you really wanted to control it from CSS like above, you would do something like this:

```
function enablePage(pageClass) {
  const pages = ['.home-page', '.products-page', '.contact-page'];
  
  for (page of pages) {
      if (page === pageClass) {
        page.classList.remove("disabled");
      } else {
        page.classList.add("disabled");
      }
  }  
}
```